app/assets/admin/css/sass/_components.scss

```diff
@@ -82,8 +82,18 @@
 				margin-top: 5px !important;
 			}
 		}
+
+		.wp-picker-input-wrap label {
```

Comment:
Wrap `label` css in `.wp-picker-input-wrap { }`

Check: https://github.com/rtMediaWP/rtMedia/pull/1405/files#diff-1a760e790d6d44ec167aaa0b34bc8c60L81

Reply:
Done